findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Reviewer.java

```diff
@@ -1,4 +1,69 @@
 package com.tp.wrc.findmyparadise.models;
 
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "reviewers")
 public class Reviewer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "reviewer_id")
+    private Integer reviewerId;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "image_src")
+    private String imageSrc;
+
+
+    @OneToMany(cascade = CascadeType.ALL)
+    @JoinColumn(name="rr_fId", referencedColumnName = "reviewer_id")
```

Comment:
I would consider renaming this column. Instead of rr_fId, maybe use fk_review_id. Also I think you mean to be referencing the review_id instead of reviewer_id.